examples/sqla/app.py

```diff
@@ -3,6 +3,7 @@
 
 from admin import app
 from admin.data import build_sample_db
+from jinja2 import StrictUndefined
```

Comment:
This is not the optimal import sorting, given that "admin" is a local folder, but ruff does not know that. I could use "." to make it clearer, or we could just be okay with it.
This seems to be the only example complex enough to have both an app.py and a folder of other modules.

Reply:
I am personally fine with it. If someone was so inclined, we could probably add some ruff config that would help it sort "correctly" - but I consider it broadly unimportant given the small scope, as you note.